internal/sql/connection.go

```diff
@@ -44,20 +60,7 @@ func NewDbConnection(cfg *Config, logger *zap.SugaredLogger) (*pg.DB, error) {
 	}
 	//--------------
 	if cfg.LogQuery {
-		dbConnection.OnQueryProcessed(func(event *pg.QueryProcessedEvent) {
-			query, err := event.FormattedQuery()
-			if err != nil {
-				panic(err)
-			}
-			logger.Infow("query time",
-				"duration", time.Since(event.StartTime),
-				"query", query)
-		})
+		dbConnection.AddQueryHook(dbLogger{})
```

Comment:
the dummy query was to ensure connection is correct.  otherwise it used to break at user query time.